WebRTC Auto Updater/Remediate - WebRTC - Remediation Script - 0.2.8.ps1

```diff
@@ -0,0 +1,275 @@
+            $preventInstall = $PreventInstallStateKey.GetValue("PreventInstallationFromMsi") 
+            if ($preventInstall -ne $null) {
+                update-log -data "Removing PreventInstallationFromMsi reg key" -Class Information -output both
+                Remove-ItemProperty -Path "HKCU:\Software\Microsoft\Office\Teams" -Name "PreventInstallationFromMsi"
+            } 
+        }
+
+        #Create a directory to save download files
+        $tempCreated = $false
+        if (!(Test-Path C:\temp)) {
+            New-Item -Path C:\ -ItemType Directory -Name temp | Out-Null
+            update-log -data "Temp path created" -output both -Class Information
+            $tempCreated = $true
+        }
+
+        # Add registry Key
+        update-log -Data "Adding IsWVDEnvironment reg key" -Class Information -output both
+        reg add "HKLM\SOFTWARE\Microsoft\Teams" /v IsWVDEnvironment /t REG_DWORD /d 1 /f /reg:64
+
+        #Download WebRTC
+        update-log -Data "Downloading Web RTC client" -Class Information -output both
+        invoke-WebRequest -Uri "https://aka.ms/msrdcwebrtcsvc/msi" -OutFile "C:\temp\MsRdcWebRTCSvc_HostSetup.msi" -UseBasicParsing -PassThru 
+ 
+        #Install MSRDCWEBTRCSVC
+        update-log -Data "Installing WebRTC client" -Class Information -output both
+        $msireturn = Start-Process msiexec.exe -ArgumentList '/i C:\temp\MsRdcWebRTCSvc_HostSetup.msi /q /n /l*voicewarmup c:\windows\temp\webrtcmsi.log' -Wait -PassThru
+        if ($msireturn.ExitCode -eq '0') {
+            update-log -data "MSIEXEC returned 0" -Class Information -Output Both
+        }
+        else {
+            $string = "MSIEXEC returned exit code " + $msireturn.ExitCode
+            update-log -data $string -Class Information -Output Both
+            exit 1
+        }
+
+        if ($tempCreated) {
+            #Remove temp folder
+            update-log -Data "Removing temp directory" -Class Information -output both
+            Remove-Item -Path C:\temp\ -Recurse | out-null
+        }
+        else {
+            #Remove downloaded WebRTC file
+            update-log -Data "Removing WebRTC client installer file" -Class Information -output both
+            Remove-Item -Path C:\temp\MsRdcWebRTCSvc_HostSetup.msi
+        }
+        #Return Success
+        update-log -Data "Media Optimization Installed" -Class Information -output both
+        return "Success"
+    }
+    catch {
+        Write-Error -ErrorRecord $_
+        return /b "Fail"
+    }
+}
+
+#function to handle user state detection logic
+function invoke-userdetect {
+    update-log -data "Detecting user state." -Class Information -output both 
+    $explorerprocesses = @(Get-WmiObject -Query "Select * FROM Win32_Process WHERE Name='explorer.exe'" -ErrorAction SilentlyContinue)
+
+    #    $explorerprocesses = @(Get-CIMInstance -Query "Select * FROM Win32_Process WHERE Name='explorer.exe'" -ErrorAction SilentlyContinue)
```

Comment:
What's the usage of this commented line here? Do we need it or can remove it?

Reply:
I missed that one in my purge of commented lines. I'll get rid of it. Thanks for the catch :)

Reply:
I removed the line and added more info into the readme.md file for the scripts. I also pushed the commit to my dev origin.